src/engraving/compat/midi/midirender.h

```diff
@@ -123,8 +114,6 @@ class MidiRenderer
 
     void collectMeasureEvents(EventMap* events, Measure const* m, const MidiRenderer::StaffContext& sctx, int tickOffset,
                               PitchWheelRenderer& pitchWheelRenderer);
-    void collectMeasureEventsSimple(EventMap* events, Measure const* m, const StaffContext& sctx, int tickOffset,
-                                    PitchWheelRenderer& pitchWheelRenderer);
     void collectMeasureEventsDefault(EventMap* events, Measure const* m, const StaffContext& sctx, int tickOffset,
```

Comment:
maybe then we can just name it "collectMeasureEvents"?

Reply:
collectMeasureEvents method already exist. So renamed collectMeasureEventsDefault to doCollectMeasureEvents. And made this method private